addons/purchase/i18n/purchase.pot

```diff
@@ -3132,3 +3130,9 @@ msgstr ""
 #: model:mail.template,subject:purchase.email_template_edi_purchase_reminder
 msgid "{{ object.company_id.name }} Order (Ref {{ object.name or 'n/a' }})"
 msgstr ""
+
+#. module: purchase
+#: model:ir.model.fields,field_description:purchase.field_res_partner__purchase_warn
+#: model:ir.model.fields,field_description:purchase.field_res_users__purchase_warn
+msgid "Purchase Order Warning"
+msgstr ""
```

Comment:
The `.pot` files are ordered alphabetically, so this needs to go after `"Purchase Order Modification used when you want [...]"`.
Note: You can see this when exporting the translations from the Settings.

Reply:
Thank you for the info
I updated it :+1: